.travis.yml

```diff
@@ -36,21 +36,30 @@
     # setuptools and pip-tools are necessary for ./travis/check_properly_generated_requirements.sh
     - pip install -U pip setuptools
     - pip install pip-tools==1.9.0
-    - pip install coveralls
-    - pip install -r dev_requirements.txt
 
   before_script:
     - psql -c "CREATE USER atmosphere_db_user WITH PASSWORD 'atmosphere_db_pass' CREATEDB;" -U postgres
     - psql -c "CREATE DATABASE atmosphere_db WITH OWNER atmosphere_db_user;" -U postgres
-    - cp ./variables.ini.dist ./variables.ini
-    - patch variables.ini variables_for_testing_${DISTRIBUTION}.ini.patch
-    - ./configure
 
   script:
     - ./travis/check_properly_generated_requirements.sh
+
+    # Test production environment
+    - pip-sync requirements.txt
+    - cp ./variables.ini.dist ./variables.ini
+    - ./configure
+    - python manage.py check
+
+    # Test development envionment
+    - patch variables.ini variables_for_testing_${DISTRIBUTION}.ini.patch
+    - ./configure
+    - pip-sync dev_requirements.txt
+    - python manage.py test --keepdb
```

Comment:
Ah. It's because you run the tests twice.

Reply:
To clarify, the `./manage.py test --keepdb` command will fail if you run it twice. Only run it once, or drop the `--keepdb`

Reply:
Mmm.. Thanks.